hack/test/libvirt/libvirt.sh

```diff
@@ -59,7 +59,7 @@ function up {
     yq w -i init.yaml cluster.network.cni.name 'custom'
     yq w -i init.yaml cluster.network.cni.urls[+] "${CNI_URL}"
     yq w -i controlplane.yaml machine.install.extraKernelArgs[+] 'console=ttyS0'
-    yq w -i join.yaml machine.install.extraKernelArgs[+] 'console=ttyS0'
```

Comment:
we should include a note in `hack/release.toml` about the change 

we can drop all the notes there and put a new one, as it will be the first one for 0.12